src/coreclr/coreclr.proj

```diff
@@ -3,11 +3,17 @@
   <Import Project="Directory.Build.targets" />
 
   <Target Name="Build">
+    <PropertyGroup>
+      <CMakeArgs>$(CMakeArgs.Replace("x#160;", " "))</CMakeArgs>
+    </PropertyGroup>
     <ItemGroup>
       <_CoreClrBuildArg Condition="$([MSBuild]::IsOsPlatform(Windows))" Include="-skiptests" />
       <_CoreClrBuildArg Condition="'$(TargetArchitecture)' != ''" Include="-$(TargetArchitecture)" />
+      <_CoreClrBuildArg Include="$(CMakeArgs)" />
       <_CoreClrBuildArg Include="-$(Configuration.ToLower())" />
+      <_CoreClrBuildArg Include="$(Compiler)" />
       <_CoreClrBuildArg Condition="'$(ContinuousIntegrationBuild)' == 'true'" Include="-ci" />
+      <_CoreClrBuildArg Condition="'$(CrossBuild)' == 'true'" Include="-cross" />
```

Comment:
[IMO]
Now that all native build scripts share common arguments via: https://github.com/dotnet/runtime/blob/e2325f35a3d92e435884326b46143e8711a261ca/eng/native/build-commons.sh#L160-L194 we can extract these argument construction from `coreclr/coreclr.proj`, `installer/corehost/build.proj` and `libraries/Native/build-native.proj` into a common `.props` file, to avoid three different approaches of doing exactly the same thing (coreclr: ItemGroup, installer: property concatenation and libraries: separate properties followed by concatenation).

Reply:
cc: @jkoritzinsky, @ViktorHofer